build.sbt

```diff
@@ -292,7 +302,8 @@ lazy val nscplugin =
       ),
       libraryDependencies ++= Seq(
         "org.scala-lang" % "scala-compiler" % scalaVersion.value,
-        "org.scala-lang" % "scala-reflect"  % scalaVersion.value
+        "org.scala-lang" % "scala-reflect"  % scalaVersion.value,
+        collectionsCompatLib
```

Comment:
We cannot depend on anything in `nscplugin` except `scala-compiler` and `scala-reflect`. Because when users use the compiler plugin, they just get its jar, and not the transitive dependencies. So this dependency must be removed.